suite/tests/CMakeLists.txt

```diff
@@ -4141,7 +4143,10 @@ if (UNIX)
   tobuild(linux.sigplain101 linux/sigplain101.c)
   tobuild(linux.sigplain110 linux/sigplain110.c)
   tobuild(linux.sigplain111 linux/sigplain111.c)
-  tobuild(linux.sigaction   linux/sigaction.c)
+  tobuild(linux.sigaction.signal_intercept   linux/sigaction.c)
```

Comment:
Not sure this needs this suffix?  I would think the "native" suffix on the new test would be sufficient.

Reply:
My thought was to suffix all the tests that need a `-no_intercept_all_signals` variant with `.signal_intercept`.  Are you suggesting using `sigaction` as the regex for the new suite in main_run_list?

Reply:
I guess I'm imagining the added test config being short-lived if we remove -no_intercept_all_signals soon and so the value of new naming seems diminished and we might want to change it back soon in any case.

Reply:
Makes sense. Removed the new config. I'm not using the test name itself (`sigaction$`) for the new suite.

Reply:
> I'm not using the test name itself (sigaction$) for the new suite.

s/not/now